release/models/aft/openconfig-aft-types.yang

```diff
@@ -97,7 +103,16 @@ module openconfig-aft-types {
       }
       enum UDP {
         description
-          "The encapsulation header is UDP packet header.";
+          "DEPRECATED. The encapsulation header is UDP packet header. This node
+          is deprecated in favor of UDPV4 and UDPV6 nodes.";
+      }
+      enum UDPV4 {
+        description
+          "The encapsulation header is UDPV4 packet header.";
```

Comment:
Suggest that we say here "The encapsulation header is a UDP + IPv4 header." given that there's not really a single "UDPV4" header. We decided that combining UDP and IPv[46] together was the right thing to do since there wasn't a use case for just a UDP header with no corresponding IP header.